README.md

```diff
@@ -66,5 +72,11 @@ categories = ['/Eletronics/','/Eletronics/Computers']
 categories = ['eletronics','eletronics-computers']
 ```
 
+### Styles API
+:construction: :construction: :construction:
+
+## Troubleshooting
+You can check if others are passing through similar issues [here](https://github.com/vtex-apps/carousel/issues). Also feel free to [open issues](https://github.com/vtex-apps/carousel/issues/new) or contribute with pull requests.
 
-See an example at [Dreamstore](https://github.com/vtex-apps/dreamstore-theme/blob/master/react/components/GalleryWrapper.js) app
+## Tests
+:construction: :construction: :construction:
```

Comment:
Update like the template, add this:

`To execute our tests go to react/ folder and run yarn test`